stdlib/public/Concurrency/Task.swift

```diff
@@ -818,6 +803,10 @@ func _taskCancel(_ task: Builtin.NativeObject)
 @_silgen_name("swift_task_isCancelled")
 func _taskIsCancelled(_ task: Builtin.NativeObject) -> Bool
 
+@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
```

Comment:
This should now be:

```suggestion
@available(SwiftStdlib 5.5, *)
```

it'll cause less conflicts etc